src/documentation/dashboard/solutions/add_domain.md

```diff
@@ -30,8 +32,11 @@ We cover the different domain parameters presented in the **Add New Domain** tab
   - You can also use a custom domain.
     - In this case, instead of having a gateway subdomain and a gateway name as your domain, the domain will be the custom domain entered in this field.
   - If you select **Custom domain**, make sure to set a DNS A record pointing to the gateway IP address on your domain name registrar.
-
-![Custom Domain Name](../img/add_domain_8.png)
+- **Supported Interfaces**
+  - You can select the network interface
+    - `WireGuard`
+    - `Planetary`
+    - `Mycelium`
 
```

Comment:
It will support all the network interfaces provided.
which mean it will depend on your choice when you created the VM, if you choose only mycelium so only mycelium will be there in the drop down menu.
if you choose all network when deployed a VM you can see all the network in the drop down menu.

Ex. for all network selected:
![Screenshot from 2024-12-11 15-50-01](https://github.com/user-attachments/assets/de9fc826-2bb7-4b41-83ad-5c8a76c74ad6)

Reply:
OK thanks for the clarification. LMK if what I just committed works now.